src/main/app/src/app/taken/taak-view/formio/formio-setup-service.ts

```diff
@@ -235,4 +242,44 @@ export class FormioSetupService {
         ),
     };
   }
+
+  private initializeAvailableDocumentsFieldsetComponent(
+    fieldsetComponent: ExtendedComponentSchema,
+  ): void {
+    if (!this.taak) {
+      return;
+    }
+
+    const documentViewComponent = fieldsetComponent.components?.find(
+      (component: { key: string }) => component.key === "ZAC_Documents",
```

Comment:
It would be nice to have a fixed set of these component keys with related `Fieldset` names and a reference to the initialize method? 

Does that make sense to you?

Reply:
Keys are the names under which the submitted data is stored. Searching for a specific key name is not a thing i would do at the moment. One of the reasons is that a form might have multiple components with the same purpose (user/group, reference table values, etc.) and keys needs to be different.